tests/test_snippets.py

```diff
@@ -14,6 +14,8 @@
 
 import compile_code
 
+RELEASE = 'RUST_RELEASE' in os.environ
```

Comment:
This will be a little bit dangerous. If we `export RUST_RELEASE=false` or `export RUST_RELEASE=0`, this `RELEASE` will still equal to `True`. It's better to check something like `'RUST_RELEASE' in os.environ and os.environ['RUST_RELEASE'] == 1` for example.